src/lib/services/group-service.ts

```diff
@@ -286,6 +289,52 @@ export class GroupService {
         }
     }
 
+    async syncExternalGroupsWithAudit(
```

Comment:
This method is a replacement of syncExternalGroups. It's only used in enterprise repo and because of that overloading the parameters was more complicated. After updating enterprise I'll remove the other method which is now marked as deprecated